source/mir/date.d

```diff
@@ -2738,6 +2665,60 @@ public:
         return Date(_julianDay - lhs);
     }
 
+    ///
+    ref Date opOpAssign(string op)(int lhs) return @safe pure nothrow @nogc
+        if (op == "+" || op == "-")
+    {
+        mixin("this._addDays(" ~ op ~ "lhs);");
```

Comment:
please avoid mixins because bad error messages.

Reply:
Addressed in latest commits. Some of the function parameters are also listed as lhs, but rhs makes more sense.